config/config-example.toml

```diff
@@ -31,6 +53,43 @@ abbreviate_modules = false
 # consensus messages.
 secret_key_path = '/etc/casper/validator_keys/secret_key.pem'
 
+# The maximum number of blocks by which execution is allowed to lag behind finalization.
+# If it is more than that, consensus will pause, and resume once the executor has caught up.
+max_execution_delay = 3
+
+
+# =======================================
+# Configuration options for Zug consensus
+# =======================================
+[consensus.zug]
+
+# Request the latest protocol state from a random peer periodically, with this interval.
+# '0sec' means it is disabled and we never request the protocol state from a peer.
+sync_state_interval = '1sec'
```

Comment:
should not this be 0, disabled

Reply:
2023-08-29 11:05:44.824000+00:00 isnt this the time for Era 10552

Reply:
Correct, I had the end Era instead of the start era and was off by one.  Corrected.